backend/unit_test.php

```diff
@@ -1,132 +1,322 @@
 <?php
 
-#---------------testing for database connection----------------- #
-include 'config.php';
-GLOBAL $db;
-print $db;
+$passed = $failed = 0;
 
-#-----------------------account creation check ---------------------#
-include 'create.php';
-
-#case1: normal user
-$create_usr1 = array($name = "Bob", $password = "123abc");
-$_REQUEST['body'] = $json_encode($create_usr1);
-$_REQUEST['type'] = "create_account";
-
-#case2: unexpected field
-$create_usr2 = array($name = "#fn4%&", $password = "234hbbtg");
-$_REQUEST['body'] = $json_encode($create_usr2);
-$_REQUEST['type'] = "create_account";
 
-#case3: missing field
-$create_usr3 = array($name = "", $password = "234hbbtg");
-$_REQUEST['body'] = $json_encode($create_usr3);
-$_REQUEST['type'] = "create_account";
+#---------------testing for database connection----------------- 
+include '../config.php';
```

Comment:
This file path does not exist. Either create a unit test subdirectory (e.g., `backend/test/unit_test.php`) or change `../config.php` to `config.php`